map/place_page_info.cpp

```diff
@@ -187,6 +187,30 @@ std::string Info::FormatSubtitle(bool withType) const
   return result;
 }
 
+std::string Info::FormatTitle() const
+{
+  std::string result;
+
+  // set result to standard name, replace the val of result for special cases.
+  result = m_primaryFeatureName;
+
+  // show local_ref for bus stops etc.
+  if (IsPublicTransportStop())
+  {
+    auto const local_ref = GetMetadata(feature::Metadata::FMD_LOCAL_REF);
+    if (!local_ref.empty())
+    {
+      //TODO: we have no way to format strings in core
+      //std::string localizedFormat = platform::GetLocalizedString("poi_title_local_ref");
+      //result = std::format(localizedFormat, m_primaryFeatureName, local_ref); cant do because not c++20
```

Comment:
```suggestion
      // TODO: we have no way to format strings in core
      // std::string localizedFormat = platform::GetLocalizedString("poi_title_local_ref");
      // result = std::format(localizedFormat, m_primaryFeatureName, local_ref); cant do because not c++20
```

1. Does std::format support positional arguments? E.g. in English, the string can be "{1} something {2}", in another lang it can be "{2} something {1}", etc.
2. Note that iOS and Android strings use different placeholders for strings: `%1$@` and `%1$s` respectively (`%@` and `%s` without positions). Format in strings.txt always uses `$@` and then converts to the necessary native format with a Twine tool. Can you reuse them? Or does it make sense to store C++ strings in their own format separately?

Reply:
1. yes it does https://www.geeksforgeeks.org/cpp-20-std-format/
2. i guess it'd make most sense to store them in their own format, either way it's the same storage usage so might as well make the difference explicit